webapi/src/main/scala/org/knora/webapi/responders/v1/SipiResponderV1.scala

```diff
@@ -160,15 +160,22 @@ class SipiResponderV1 extends ResponderV1 {
             conversionResultResponse <- recoveredConversionResultFuture
 
             // get file type from Sipi response
-            responseAsMap: Map[String, JsValue] = conversionResultResponse.entity.asString.parseJson.asJsObject.fields.toMap
+            responseAsMap: Map[String, JsValue] = conversionResultResponse.entity.asString.parseJson.asJsObject.fields
 
             statusCode: Int = responseAsMap.getOrElse("status", throw SipiException(message = "Sipi did not return a status code")) match {
                 case JsNumber(ftype: BigDecimal) => ftype.toInt
```

Comment:
I have just realized that `BigDecimal` does not have anything to do with Sipi's behaviour, but with `spray.json`. I (wrongly) assumed that we did this because Sipi used to return floating point numbers for all numeric values although they might be integers in Lua. See: https://github.com/dhlab-basel/Sipi/pull/65